LuckySpin/Controllers/SpinnerController.cs

```diff
@@ -89,12 +68,9 @@ public IActionResult Index(Player player)
                 ViewBag.Display = "block";
             else
                 ViewBag.Display = "none";
-            //TODO Assign a ViewBag.PlayerId item used to assigns a link its route_id in SpinIt View
-            //      (see the <a href> for "Current Balance" in the SpinIt.cshtml file)
 
+            ViewBag.PlayerId = idNum;
 
-            //TODO Compare DB records when adding a generic Spin, as shown below, 
-            //     with adding a new Spin to the current player's Spins list
             _dbc.Spins.Add(new Spin { IsWinning = spinVM.IsWinning });
```

Comment:
Here is where you are missing the difference between adding a Spin to the database,
and adding a Spin to the currentPlayer's Spin list  property using the dot syntax,

currentPlayer.Spins.Add(...)